### DISCUSSION TITLE
(Context,Search Query) Dataset

### DISCUSSION THREAD

**Discussion by atharvjairath** [facebookresearch/ParlAI#4137] [Q&A] [answered]
Hi, 
I actually wanted to know is the (Context,Search Query) dataset available from the paper **Internet-Augmented Dialogue Generation** or not?
I tried Wizard of Internet dataset but it was not showing the Search Query based on Context.

**Example :**
```
- - - NEW EPISODE: WizInternetWizardTeacher - - -
My favorite hobby is Crochet.
Creating blankets for veterans
   How many blankets have you crocheted for veterans since you started your hobby?
I'm working on number 23 right now. Do you Crochet?
   I am unable to crochet but find it interesting.  How did you get started in crochet?
My mother in law taught me. To pass the time. 
   Do you have a favorite crochet pattern?
```

Arguments passed:
`parlai display_data --task wizard_of_internet`

I am not sure but there should be an parameter I can pass with this argument to get my desired result.
`--display-add-fields`

**Comment by mojtaba-komeili** [comment, marked as answer]
There are multiple teachers in that task. The command that you used shows the default one, that is Wizard's dialog response. If you want to see the search query for that context try `parlai display_data --task wizard_of_internet:SearchQueryTeacher`.
